image/templates/helm/stackrox-secured-cluster/templates/sensor-scc.yaml

```diff
@@ -44,51 +41,53 @@ volumes:
   - emptyDir
   - secret
 
-{{- else if eq ._rox.env.openshift 4 }}
-
-{{- if false }}
-# "fake" document separator to aid GVK extraction heuristic
 ---
+
 {{- end }}
 
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: use-anyuid-scc
+  name: use-sccs-for-sensor
   namespace: {{ ._rox._namespace }}
   labels:
-    {{- include "srox.labels" (list . "role" "use-anyuid-scc") | nindent 4 }}
+    {{- include "srox.labels" (list . "role" "use-sccs-for-sensor") | nindent 4 }}
   annotations:
-    {{- include "srox.annotations" (list . "role" "use-anyuid-scc") | nindent 4 }}
+    {{- include "srox.annotations" (list . "role" "use-sccs-for-sensor") | nindent 4 }}
 rules:
 - apiGroups:
   - security.openshift.io
   resources:
   - securitycontextconstraints
   resourceNames:
+{{- if and ._rox.env.openshift ._rox.system.createSCCs }}
+  - stackrox-sensor
```

Comment:
I guess this can be made unconditional as well? But I'm not sure if it should

Reply:
I left it conditional for now, let me know if we should remove the `if`

Reply:
With the `users` section coming back, I guess this can be removed? It will make upgrade instructions easier